docs/semgrep-secrets/conceptual-overview.md

```diff
@@ -0,0 +1,111 @@
+
+* **Line 2:** Semgrep and regex-based scanners can detect both the variable name `secret` and its value (token) in line 2.
+	* A regex-based scanner may generate a noisy finding from line 2 even though `secret` has not been passed to any function.
+	* Semgrep doesn't generate a finding because the token hasn't been passed as a `secretAccessKey` or similar.
+* **Line 7:** Semgrep and regex-based scanners are able to detect **line 6**, in which the plain-text secret is passed to the `AWS.config.update` function.
+* **Line 17:** Semgrep and regex-based scanners can detect **line 14**, in which `secret` is passed.
+* **Line 26:** Semgrep correctly skips `conf.secret` in **line 21**. Regex-based scanners simply looking for matches of the string `secret` generate a false positive.
+
+After scanning for secrets, Semgrep uses a **post-processor** function called a **validator** to validate a secret.
+
+1. The post-processor detects the service, such as Slack or AWS, that the secret is used for.
+2. If the post-processor does not support the service that the secret is used for, or the secret is a **honeypot token**, Semgrep notes that there is **No validator** for the secret.
+3. If the validator can detect the service, Semgrep performs an API call. The following outcomes can occur:
+	1. If the API call returns an HTTP response of 200 or similar, then the finding's validation state is **confirmed valid**.
+	2. If the API call returns an HTTP response of 403 or similar, then the finding's validation state is **confirmed invalid**.
+	3. If the API call returns an HTTP response of 500 or similar, or some other error occurred, then Semgrep notes that a **validation error** has occurred.
+
+All findings, whether they are validated, with validation errors, invalid, or valid, appear in Semgrep Cloud Platform.
+
+By performing this post-processor check, you are able to prioritize (triage) the most high priority, active findings.
+
+<!--
+:::note
+For a list of all supported services that Semgrep can detect, see Semgrep post-processor list.
+:::
+-->
+
+## Fine-tuning findings through entropy analysis
+
+In secret scanning, entropy is the measure of a **string's randomness**. It is used to measure how likely a string is random. If a string is highly entropic, it is highly random, and therefore likelier to be a secret. By performing entropy analysis, Semgrep is able to reduce false positives.
+
+Examples of high-entropy (random) strings:
+
+```
+VERZVs+/nd56Z+/Qxy1mzEqqBwUS1l9D4YbqmPoO
+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
+```
+
+Examples of low-entropy strings:
+
+```
+XXXXXX
+txtCfmPassword
+```
+
+## Differences between Semgrep Secrets and Semgrep Registry rules
+
+The Semgrep Registry includes SAST rules that can detect secrets to a certain extent. You can run these rules in Semgrep Code (Semgrep's SAST analyzer), or even write your own custom secret-detecting SAST rules, but with the following differences:
+
```

Comment:
I would also add here that existing Secrets coverage in SAST is being transitioned to Secrets only (OSS rules will remain OSS)